packages/smooth_app/lib/l10n/app_es.arb

```diff
@@ -1,5 +1,5 @@
 {
-    "app_name": "OpenFoodFacts",
+    "app_name": "OpenFoodFactsEs",
```

Comment:
?

Reply:
It seems it's confusing it with the Twitter account strings